app/_includes/info_box/sections/topologies.html

```diff
@@ -1,8 +1,19 @@
 <div class="flex flex-col gap-2  w-full">
-  <div class="text-primary font-medium">Supported Topologies</div>
+  <div class="text-primary font-medium">Supported Gateway Topologies</div>
   <div class="flex gap-1.5 flex-col md:flex-row">
-    {% for topology in include.topologies %}
+    {% for topology in include.topologies.on_prem %}
     <span {% unless forloop.last %}class="after:content-[',']"{% endunless %}>{{ topology }}</span>
     {% endfor %}
   </div>
-</div>
\ No newline at end of file
+</div>
+
+{% if include.topologies.konnect_deployments %}
+<div class="flex flex-col gap-2  w-full">
+  <div class="text-primary font-medium">Supported Konnect Deployments</div>
```

Comment:
```suggestion
  <div class="text-primary font-medium">Konnect Deployments</div>
```
Same comment here if you want to shorten it, but I think it's fine as is.